swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/TunnelStore.swift

```diff
@@ -123,10 +123,7 @@ public final class TunnelStore: ObservableObject {
   func createManager() async throws {
     let protocolConfiguration = NETunnelProviderProtocol()
     let manager = NETunnelProviderManager()
-    let providerConfiguration =
-      protocolConfiguration.providerConfiguration
-      as? [String: String]
```

Comment:
This was the bug.

Reply:
Was it trying to read something that wasn't there?

Reply:
I think by default it's an empty dict `[]` which can be cast to `[String: String]`, causing the conditional to evaluate `true`.